app/src/main/java/com/infomaniak/drive/ui/login/LoginActivity.kt

```diff
@@ -96,11 +95,12 @@ class LoginActivity : AppCompatActivity() {
                     signInButton.isInvisible = !showConnectButton
                 }
             })
+
+            nextButton.setOnClickListener { currentItem++ }
         }
 
-        dotsIndicator.setViewPager2(introViewpager)
+        dotsIndicator.attachTo(introViewpager)
 
-        nextButton.setOnClickListener { introViewpager.currentItem = introViewpager.currentItem + 1 }
 
```

Comment:
Why did you do this?
And what does this have to do with RA?

Reply:
I did this because setViewPager2() is deprecated and must be replace with attachTo(), and  the listener used 2 times introViewPager to update one of its attributes and there is already an introViewPager.apply {} right above (ending line 100).

It could have been in another PR but I kept it in this one because it's in the same function that I was already modifying and it's a small change. I can deplace it in another PR if needed.

Reply:
done in #925